### ISSUE TITLE
Presence in Stackage

### ISSUE THREAD

**Issue by varosi** [haskell/haskell-ide-engine#328]
It'll be really nice if it's presented in LTS in Stackage:
https://www.stackage.org/

This will make installation even easier on major OSes without source download. Just:

> stack install haskell-ide-engine

and Voila!

**Comment by alanz** [comment]
I agree

But there are a number of dependencies that are not yet on stackage, and I am waiting for them.

Most notably are `cabal-helper` and `ghc-mod`, which are still in the process of being updated for GHC 8.2.1 and Cabal 2.0, which brought some pretty big changes.

**Comment by varosi** [comment]
Great!

**Comment by alanz** [comment]
This is currently blocked by not having dependencies on stackage.

**Comment by l29ah** [comment]
Are you going to make a proper fork of ghc-mod, since the @DanielG is unable to keep up with the pull requests for a few years already?

**Comment by expipiplus1** [comment]
@alanz Are these dependencies also missing from Hackage?

**Comment by alanz** [comment]
The dependencies is what is keeping hie off hackage. I am hoping that by the end of GSOC this situation will be resolved this year.

**Comment by Ciantic** [comment]
@alanz just inquiring. Since you posted your comment *before GSOC started*. However now that I look at this year GSOC, I don't think any of the GSOC projects are related? So this effort is still largely waiting?

**Comment by poscat0x04** [comment]
Any updates on this?

**Comment by alanz** [comment]
@poscat0x04 We are not intending to put haskell-ide-engine on hackage.

This is mainly because it has been superseded by [haskell-language-server](https://github.com/haskell/haskell-language-server), and publishing hie would set expectations that it would be maintained.

So we are going to ask people to be patient some more. Sorry.

**Comment by varosi** [comment]
Yes, understandable!

Thanks for all the effort on the project!
